site/content/blogs/release-v126.en.md

```diff
@@ -36,6 +36,91 @@ Copilot v1.26 brings several new features and improvements:
 
 ## `storage init` for environment addons
 
+Previously, `copilot storage init` only supports storage addon that is attached to your workload: 
```

Comment:
```suggestion
Previously, `copilot storage init` only supported storage addons attached to workloads: 
```

Reply:
Also, can we take out the newline to match the "Now" section?

Reply:
<img width="775" alt="Screenshot 2023-02-16 at 4 59 56 PM" src="https://user-images.githubusercontent.com/79273084/219507130-87184c19-463b-4323-805f-de1f4d29e23a.png">

The newline isn't rendered as a new line! It takes two consecutive newline to render a real new line.